### PR TITLE
docs(observables): Add docs for operator : creation - timer

### PR DESCRIPTION
**Pull request by cschomer**
Add docs for operator : creation - timer #84

### CODE REVIEW

File: src/operator-docs/creation/timer.ts [hunk 1]
```diff
@@ -1,6 +1,65 @@
 import { OperatorDoc } from '../operator.model';
 
 export const timer: OperatorDoc = {
-  'name': 'timer',
-  'operatorType': 'creation'
+  name: 'timer',
+  operatorType: 'creation',
+  signature:
+    'public timer(initialDelay: number | Date, period : number = 0, scheduler: IScheduler = async): Observable<number>',
```
Comment (JWO719) [inline src/operator-docs/creation/timer.ts]: could you remove the generic as described in the contribution.md

File: src/operator-docs/creation/timer.ts [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+  marbleUrl: 'http://reactivex.io/rxjs/img/timer.png',
+  shortDescription: {
+    description: `Creates an Observable that starts emitting after an initialDelay
+    and emits ever increasing numbers after each period of time thereafter`
+  },
+  walkthrough: {
+    description: `timer returns an Observable that emits an infinite sequence of ascending integers,
+    with a constant interval of time, period of your choosing between those emissions.
+    The first emission happens after the specified initialDelay.
+    The initial delay may be a Date. By default,
```
Comment (JWO719) [inline src/operator-docs/creation/timer.ts]: In the signature you also mention number? Does this refer to an unix timestamp? In this case you could add a sentence for this case, to make it a bit clearer.

File: src/operator-docs/creation/timer.ts [hunk 3]
```diff
@@ -1,6 +1,65 @@
...
+    If period is not specified, the output Observable emits only one value, 0.
+    Otherwise, it emits an infinite sequence`
+  },
+  examples: [
+    {
+      name:
+        'Emits ascending numbers, one every second (1000ms), starting after 3 seconds running for 10 seconds',
+      code: `
+          import { timer } from 'rxjs/observable/timer'
+          import { pipe } from 'rxjs';
```
Comment (JWO719) [inline src/operator-docs/creation/timer.ts]: pipe doesn't need to be imported?!

File: src/operator-docs/creation/timer.ts [hunk 4]
```diff
@@ -1,6 +1,65 @@
...
+          import { take } from 'rxjs/operators';
+          const observable = timer(3000, 1000).pipe(take(10));
```
Comment (JWO719) [inline src/operator-docs/creation/timer.ts]: could you add a $ suffix to the variable name, to stick to conventions. And there could be a better variable name as "observable", for exmple "timer$"

File: src/operator-docs/creation/timer.ts [hunk 5]
```diff
@@ -1,6 +1,65 @@
...
+          const subscription = observable.subscribe({
+            next: (n) => console.log(n), // triggered every 1000ms as specified from period param
+            complete: () => console.log('complete')
+          });
```
Comment (JWO719) [inline src/operator-docs/creation/timer.ts]: Could you add the expected output as comments. I know it isn't really special in this case...

**Comment by JWO719** [comment]
And could you add the issue number in the comment at top, so that the issue is closed when the pr is merged.

File: src/operator-docs/creation/timer.ts (hunk 1 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Removed in recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

File: src/operator-docs/creation/timer.ts (hunk 2 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Added in recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

File: src/operator-docs/creation/timer.ts (hunk 3 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Yep removed in recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

File: src/operator-docs/creation/timer.ts (hunk 4 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Added recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

File: src/operator-docs/creation/timer.ts (hunk 3 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Removed recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

File: src/operator-docs/creation/timer.ts (hunk 5 above)
Comment (cschomer) [reply src/operator-docs/creation/timer.ts]: Added in recent commit [46b699f](https://github.com/ReactiveX/rxjs-docs/pull/299/commits/46b699fa2faee3c5cc58bf21c52aa5eebd449fd7)

**Comment by ladyleet** [comment]
@cschomer can you resolve conflicts too? @JWO719 what do you think of the new changes?

**Comment by ladyleet** [comment]
Thx @ashwin-sureshkumar! :) @cschomer just rebase and we can merge

**Comment by JWO719** [comment]
Woop Woop, Great job !!

**Pull request merged**